pkg/controller/operators/catalog/operator.go

```diff
@@ -85,6 +87,7 @@ type Operator struct {
 	catalogSubscriberIndexer map[string]cache.Indexer
 	clientAttenuator         *scoped.ClientAttenuator
 	serviceAccountQuerier    *scoped.UserDefinedServiceAccountQuerier
+	bundleLoader             *configmap.BundleLoader
```

Comment:
I don't know, Abu actually wrote the v1 BundleLoader in operator-registry. I assume it was just modeled after the sqlite loader.

Reply:
I think in an ideal world this would be an interface. It would be nice if we had other options than unpacking into a configmap in the future. But we can do that as a separate refactoring after this merges.